microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
 
-<!-- terminology hit counts as of 1/26/2022:
-audit: 93 hits
-lighthouse: 9 -->
-This tutorial uses the **Lighthouse** tool.  The **Lighthouse** tool provides links to content hosted on third-party websites.  Microsoft is not responsible for and has no control over the content of these sites and any data that may be collected.
-
-The **Lighthouse** tool was previously called the **Audits** tool.  "Audits panel" means the same thing as "Lighthouse tool".
+The **Lighthouse** tool provides links to content hosted on third-party websites.  Microsoft isn't responsible for and has no control over the content of these sites and any data that may be collected.
 
 
 <!-- ====================================================================== -->
 ## Prerequisites
 
-*  You should have basic web development experience, similar to what is taught in this [Introduction to Web Development class](https://www.coursera.org/learn/web-development#syllabus).
+* You should have some web development experience, similar to what is taught in this [Introduction to Web Development class](https://www.coursera.org/learn/web-development#syllabus).
+
+* Install [Visual Studio Code](https://code.visualstudio.com) to edit source code.
+
+* Install [Node.js](https://nodejs.org) to use it as a local web server.
 
-*  You don't need to know anything about load performance.  You learn about load performance in this tutorial.
+* You don't need to know anything about load performance.  You learn about load performance in this tutorial.
 
 
 <!-- ====================================================================== -->
 ## Introduction
 
-This is Tony.  Tony is very famous in cat society.  He has built a website so that his fans can learn about his favorite foods.  His fans love the site, but Tony keeps hearing complaints that the site loads slowly.  Tony has asked you to help him speed the site up.
+In this tutorial, you improve the performance of [Margie's travel](https://microsoftedge.github.io/Demos/travel-site/), a fictitious travel website that contains travel images, text descriptions, a few JavaScript-based user interactions, and an interactive map.
 
-![Tony the cat](../media/speed-tony.msft.png)
+The source files for the website are at [MicrosoftEdge / Demos > travel-site](https://github.com/MicrosoftEdge/Demos/tree/main/travel-site). 
 
 
 <!-- ====================================================================== -->
-## Step 1: Audit the site
+## Step 1: Set up the website locally
+
+First, set up the website locally so that you can make changes to it later:
+
+1. Get the website's source code locally: [Download or clone the Demos repo](../sample-code/sample-code.md#download-or-clone-the-demos-repo).
+
+1. Open the folder you just downloaded or cloned in Visual Studio Code.
+
+1. In Visual Studio Code, click **View** > **Terminal** or press `Ctrl` + `` ` ``.
+
```

Comment:
is it better advice to not use Terminal, but use sep shell window?  If use Terminal inside VS Code, will need to restart web server every time you close VS Code window.  Might be ok here.  

Maybe mention that starting server from within a shell window gives more long-term running of the server.

Reply:
I think it's ok here. I don't expect the case where someone starts reading the docs, and then continues over multiple days, sometimes closing VS Code to be common. I think it's more likely that readers will spend a minimal amount of time on the docs, setting things up, and then using one of the sections only.

The other advantage I see is that we don't need people to start another app, or even know about external terminal apps at all. VS Code gives them everything they need.